components/Buttons/src/MDCFloatingButton.h

```diff
@@ -96,6 +140,60 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
 
 - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_DESIGNATED_INITIALIZER;
 
+- (void)setMinimumSize:(CGSize)size NS_UNAVAILABLE;
+
+/**
+ Sets the minimum size when the button has the specified @c shape @c mode.
+ Setting a size of @c CGSizeZero is equivalent to no minimum size.
+
+ @param minimumSize The new minimum size of the button.
+ @param shape The shape that the size constrains.
+ @param mode The mode that the size constrains.
+ */
+- (void)setMinimumSize:(CGSize)minimumSize
```

Comment:
We should elaborate that this is how you change the size of the default 56 and mini 40 buttons.

Reply:
Updated description of #2566 to include this request.